Update for recent crafter capabilities changes. (#400)
ldtteam/MinecoloniesWiki: source/buildings/bakery.md

On page two of the GUI, you will see a list of recipes that the Baker is able to make. Just click to change the ones you want the Baker to bake to on and they will begin making those items as long as they receive the required ingredients. (The black box at the top is to search for recipes.) The bakery already knows the recipes for bread, cakes, cookies, and pumpkin pies without you having to teach them. 

> You can still teach the Baker those recipes to enable them to be made on demand, such as when the [Builder](../../source/workers/builder) needs cake as a decoration. 

Reviewer: ```suggestion
You can still teach the Bakery those recipes to enable them to be made on demand, like if the [Builder](../../source/workers/builder) needs cake as a decoration. 
```
